lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_merge_translators_strings.rb

```diff
@@ -9,7 +9,7 @@ class AndroidMergeTranslatorsStringsAction < Action
       def self.run(params)
         folder_path = File.expand_path(params[:strings_folder])
 
-        subfolders = Dir.entries("#{folder_path}")
+        subfolders = Dir.entries(folder_path.to_s)
```

Comment:
I'm assuming Rubocop added `.to_s`. It's not explicit (yet, maybe in Ruby 3.x at some point?) but [`File.expand_path` returns a string](https://www.rubydoc.info/stdlib/core/File.expand_path) at least as far as one can see from the examples, so I think we could simplify to 
```suggestion
        subfolders = Dir.entries(folder_path)
```

Reply:
Indeed this was added by auto correct but you're right that they should not be needed, will remove.